runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -1961,6 +1961,376 @@ J9::Z::TreeEvaluator::inlineUTF16BEEncode(TR::Node *node, TR::CodeGenerator *cg)
    return node->setRegister(translated);
    }
 
+TR::Register*
+J9::Z::TreeEvaluator::inlineCRC32CUpdateBytes(TR::Node *node, TR::CodeGenerator *cg, bool isRawAddress)
+   {
+   // Get call parameters
+   TR::Register* crc = cg->gprClobberEvaluate(node->getChild(0));
+   TR::Register* arr = cg->gprClobberEvaluate(node->getChild(1));
+   TR::Register* off = cg->gprClobberEvaluate(node->getChild(2));
+   TR::Register* end = cg->gprClobberEvaluate(node->getChild(3));
+
+   // Registers
+   TR::Register* tablePtr = cg->allocateRegister();
```

Comment:
Can we allocate register closer to the point it is used first also similar comment for other variables used within the evaluators. 

Also `tablePtr` seems like a generic name, it is used to store the address of the table containing CRC32C Constants, right? Should reflect that.